README.md

```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# To Do List (Structure)
+# Leader Board (API)
```

Comment:
- [x] Good Job at implementing this change suggested by the reviewer, but as he said you had to make sure your readme file has a correlation with your project and when we click on your live demo section, it shows me a page of 404 saying this page does not exist, in this case, we recommend to try to deploy it again and if that doesn't work, to remove this section of the live link from the project to avoid misunderstandings with the people who check on your project.

```
## Live Demo
[Live Demo Link](https://oyelaking9.github.io/todoList/dist/)
```

Reply:
Alright. I'll make the changes.
Thanks 👌✍

Reply:
@oyelakinG9  problem :) is one step for approval ✔💪

Reply:
But we were not told to deploy yet.
Kindly tell me the link I can use 👏

Reply:
@oyelakinG9 yes, that's why I suggested in my first comment to just remove that section to avoid confusion from other people, in other project they will explain how to deploy in Heroku and Netfily which is a whole project to do ahead.

Reply:
Ok. Thanks

Reply:
@oyelakinG9  no problem, deployment was not mandatory in this project, that's why is a good suggestion to if you can't deploy it properly, then is okay to remove this section, on other projects there will be mandatory deployments, but not on this one, so you don't have to worry about that. 👍😁♻

Reply:
I've removed it